src/components/singlebook.js

```diff
@@ -23,22 +23,27 @@ const Singlebook = ({
         <p className="category">{category}</p>
         <h2 className="bookname">{title}</h2>
         <h3 className="authorname">{author}</h3>
-        <div className="acctionbuttons">
-          <button type="button">Comments</button>
-          <button type="button" onClick={handleRemoveBook}>Remove</button>
-          <button type="button">Edit</button>
+        <div className="actionbuttons">
+          <button type="button" className="comment">Comments</button>
+          <button type="button" className="remove" onClick={handleRemoveBook}>Remove</button>
+          <button type="button" className="edit">Edit</button>
         </div>
       </div>
 
       <div className="bookstatus">
-        <i className="fa-solid fa-spinner" />
-        <h4>75% completed</h4>
+        <div className="iconspinner">
+          <i className="fa-solid fa-spinner" />
+        </div>
+        <div className="progress">
+          <h4>75%</h4>
```

Comment:
- [ ] Instead of using an icon for the `circular-progressbar` I'll suggest you use the appropriate `react-circular-progressbar` component so that your design could look exactly the same as the one provided on [Zeplin](https://zeplin.io/). Kindly, take a look at this [article](https://www.npmjs.com/package/react-circular-progressbar) to see how to install and implement it in your project.